server/backend/api/app/models/model.py

```diff
@@ -667,8 +674,80 @@ class FamAccessControlPrivilege(Base):
     )
-    role = relationship("FamRole", back_populates="fam_access_control_privilege", lazy="joined")
-    user = relationship("FamUser", back_populates="fam_access_control_privilege", lazy="joined")
+    role = relationship(
+        "FamRole", back_populates="fam_access_control_privilege", lazy="joined"
+    )
+    user = relationship(
+        "FamUser", back_populates="fam_access_control_privilege", lazy="joined"
+    )
 
     def __repr__(self):
-        return f"FamAccessControlPrivilege(user_id={self.user_id}, role_id={self.role_id})"
\ No newline at end of file
+        return (
+            f"FamAccessControlPrivilege(user_id={self.user_id}, role_id={self.role_id})"
+        )
+
+
+class FamUserTermsConditions(Base):
+    __tablename__ = "fam_user_terms_conditions"
+    __table_args__ = (
+        ForeignKeyConstraint(
+            ["user_id"],
+            ["app_fam.fam_user.user_id"],
+            name="reffam_user_terms_conditions_user",
+        ),
+        PrimaryKeyConstraint(
+            "user_terms_conditions_id", name="fam_user_terms_conditions_pk"
+        ),
+        UniqueConstraint("user_id", "version_id", name="fam_tc_user_version_uk"),
+        Index("ix_app_fam_fam_user_terms_conditions_user_id", "user_id"),
+        {
+            "comment": "User Terms Conditions records identify the users who accept the "
+            "terms and conditions, as well as the version of it.",
+            "schema": "app_fam",
+        },
+    )
+
+    user_terms_conditions_id = Column(
+        BigInteger,
+        Identity(
+            start=1,
+            increment=1,
+            minvalue=1,
+            maxvalue=9223372036854775807,
+            cycle=False,
+            cache=1,
+        ),
+        comment="Automatically generated key used to identify the uniqueness of a terms and conditions acceptance record.",
+    )
+    user_id = Column(
+        BigInteger,
+        nullable=False,
+        comment="Unique ID to reference and identify the user within FAM system.",
+    )
+    version_id = Column(
+        Integer,
+        nullable=False,
+        comment="Number to identity the version of the terms and conditions the user accepted.",
+    )
+    create_user = Column(
+        String(100),
+        nullable=False,
+        comment="The user or proxy account that created the record.",
+    )
+    create_date = Column(
+        TIMESTAMP(timezone=True, precision=6),
+        nullable=False,
+        default=datetime.datetime.utcnow,
+        comment="The date and time the record was created.",
+    )
+    update_user = Column(
+        String(100),
+        comment="The user or proxy account that created or last updated the record.",
+    )
+    update_date = Column(
+        TIMESTAMP(timezone=True, precision=6),
+        default=datetime.datetime.utcnow,
+        comment="The date and time the record was created or last updated.",
+    )
+
+    user = relationship("FamUser", back_populates="fam_user_terms_conditions")
```

Comment:
I feel maybe we don't need the detail user information, so didn't put the lazy joined option

Reply:
Yes, thanks. I think we want "lazy", the default is  lazy='select'. I am still confused with `lazy='joined'` for existing objects, but I think they are "eager" loading so removing that make sense to me.